### DISCUSSION TITLE
Commands are unavailable

### DISCUSSION THREAD

**Discussion by osenvosem** [rest-nvim/rest.nvim#182] [Q&A]
I'm using LunarVim. Inserted plugin configuration from the example with corresponding corrections for the lazy.nvim. The plugin is installed with no errors accrued. But there are no rest.nvim commands available. Can't understand why.

<img width="359" alt="Screenshot 2023-02-23 at 22 25 44" src="https://user-images.githubusercontent.com/6762673/221009866-05e3ecbb-6b6b-4a1b-ba3d-1bd751d4c094.png">

**Comment by k4-100** [comment]
same issue here.
What version of lvim do you run?

**Comment by k4-100** [comment]
Aight, I found the way.
You need to add the following to your config.lua:

```
lvim.builtin.which_key.mappings["r"] = {
	name = "Rest requests",
	t = { "<Plug>RestNvim<cr>", "run the request" },
	w = { "<Plug>RestNvimPreview<cr>", "preview cUrl command" },
	d = { "<Plug>RestNvimLast<cr>", "re-run request" },
}
```

for some reasons `w` does nothing in my case, but I can run requests so it's good
